pages/partials/navTabs/administrator.ejs

```diff
@@ -0,0 +1,25 @@
+<%- include('navTab', {
+    activeSubPage:  'administrators',
+    urlSuffix:      '/administrator/admins',
+    iconClasses:    'fas fa-user-shield',
+    tabLabel:       'Administrators'}); %>
+<%- include('navTab', {
+    activeSubPage:  'courses',
+    urlSuffix:      '/administrator/courses',
+    iconClasses:    'fa fa-chalkboard',
+    tabLabel:       'Courses'}); %>
+<%- include('navTab', {
+    activeSubPage:  'networks',
+    urlSuffix:      '/administrator/networks',
+    iconClasses:    'fas fa-network-wired',
+    tabLabel:       'Exam Networks'}); %>
+<%- include('navTab', {
+    activeSubPage:  'queries',
+    urlSuffix:      '/administrator/queries',
+    iconClasses:    'fas fa-database',
+    tabLabel:       'Queries'}); %>
+<%- include('navTab', {
+    activeSubPage:  'settings',
+    urlSuffix:      '/administrator/settings',
+    iconClasses:    'fas fa-cog',
+    tabLabel:       'Settings'}); %>
```

Comment:
This tab doesn't activate for me when on the Settings page.

Reply:
Fixed in 2d9f83b.